nova/virt/vmwareapi/host.py

```diff
@@ -90,45 +91,58 @@ def update_status(self):
         """Update the current state of the cluster."""
         data = {}
         try:
-            capacity, freespace = _get_ds_capacity_and_freespace(self._session,
-                self._cluster, self._datastore_regex)
+            capacity, free, max_free = _get_ds_capacity_and_freespace(
+                self._session, self._cluster, self._datastore_regex)
 
             # Get cpu, memory stats from the cluster
-            stats = vm_util.get_stats_from_cluster(self._session,
-                                                   self._cluster)
+            per_host_stats = vm_util.get_stats_from_cluster_per_host(
+                self._session, self._cluster)
         except (vexc.VimConnectionException, vexc.VimAttributeException) as ex:
             # VimAttributeException is thrown when vpxd service is down
             LOG.warning("Failed to connect with %(node)s. "
                         "Error: %(error)s",
-                        {'node': self._host_name, 'error': ex})
+                        {'node': self._cluster_node_name, 'error': ex})
```

Comment:
Isn't this misleading? We never connect to the ESXi directly, but only to the VC.

Reply:
Yes, but the `_cluster_node_name` (previously `_host_name`) is neither the ESXi host-name nor the hostname of the VCenter.  It should be clear by the printed error.